Simplified/AxisService/NYPLAxisXML.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  AxisXML.swift
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-06.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+struct NYPLAxisXML: CustomStringConvertible {
```

Comment:
@ErnestFan @ettore  is it obvious enough as to why this class was needed? I am of the opinion that this class lacks proper documentation.

Reply:
Yeah, if you can add a few lines to explain the difference between this and NYPLXML, that would be great.
I am thinking if we need some unit tests as well, let's see what @ettore suggests.

Reply:
done